doc/2/guides/introduction/general-purpose-backend/index.md

```diff
@@ -190,18 +190,18 @@ Or start to develop a client application by using one of our [SDKs](/sdk):
  - [Node.js](/sdk/js/7/getting-started/node-js)
  - [Java](/sdk/jvm/1/getting-started/java)
  - [Kotlin](/sdk/jvm/1/getting-started/kotlin)
- - [C# .NET Core](/sdk/csharp/2/getting-started/standalone/)
+ - [C# .NET Core](https://docs.kuzzle.io/sdk/csharp/2/getting-started/standalone/)
```

Comment:
That's not the way to solve this deadlink issue: our documentation system is capable of inferring URLs as long as they point to our documentation itself.

But there is a trick you need to know: for it to work, you cannot put a trailing slash at the end of an URL. I don't remember why, I think that's a known issue with how vuesomething processes URLs.

That's why the Node, java or kotlink links work, and why the C#, react, vue and others don't.

So you should do this instead, and apply that change to other links below:

```suggestion
 - [C# .NET Core](/sdk/csharp/2/getting-started/standalone)
```

Reply:
Ok thanks for the trick, I wouldn't have guessed it otherwise :+1: